packages/backend/src/handlers/actions/3Box/cache3BoxProfile/handler.ts

```diff
@@ -1,8 +1,8 @@
 import { Request, Response } from 'express';
 
-import { updateVerifiedAccounts } from './updateVerifiedAccounts';
+import { updateCachedProfile } from './updateCachedProfile';
 
-export const updateBoxProfileHandler = async (
+export const cache3BoxProfileHandler = async (
```

Comment:
We should start using "IDX" instead of 3box for naming

Reply:
Done.